<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Amir</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QWidget" name="matplotlib_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>651</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="plot_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>XW Zar</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>متوقف</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rec">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>recorder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kmax_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kmax</string>
    </property>
   </widget>
   <widget class="QLabel" name="plot_label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>XW Zar</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Layer">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="freq">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="file_actionStart"/>
    <addaction name="separator"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="file_actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
